tests/php/Extension/FluentVersionedExtensionTest.php

```diff
@@ -206,4 +211,164 @@ public function testStagesDifferInLocale(): void
         // No Live version
         $this->assertTrue($page->stagesDifferInLocale());
     }
+
+    /**
+     * Tests various Versioned methods correctly identify versions in different locales
+     */
+    public function testCorrectVersionIdentified(): void
+    {
+        $recordID = null;
+        // Create a new record in the base record only, no localisation
+        FluentState::singleton()->withState(function (FluentState $newState) use (&$recordID) {
```

Comment:
```suggestion
        FluentState::singleton()->withState(function (FluentState $state) use (&$recordID) {
```
[nitpick] Simplify variable name.